common/utils/utils_test.go

```diff
@@ -792,3 +792,33 @@ func TestGetFQDNByIP(t *testing.T) {
 		assert.Empty(t, fqdn, "Expected empty FQDN for non-resolvable IP")
 	})
 }
+
+func TestTrimVolumePath(t *testing.T) {
```

Comment:
How about adding /volume and //volume, some paths with _ in the name.

Reply:
addressed